client/index.html

```diff
@@ -241,29 +241,28 @@ <h1 class="title">Settings</h1>
 							<div class="col-sm-12">
 								<h2>Notifications</h2>
 							</div>
-							<div class="col-sm-12">
+							<div class="col-sm-6">
 								<label class="opt">
 								<input id="badge" type="checkbox" name="badge">
 								Enable badge
 								</label>
-							</div>
-							<div class="col-sm-12">
```

Comment:
I've changed it back to what it was before and it works like I would have expected it to, which is odd. So I've reverted this.